src/daemon/plugin.c

```diff
@@ -80,6 +80,23 @@ struct write_queue_s
 	write_queue_t *next;
 };
 
+#define FF_SIMPLE  0
+#define FF_REMOVE  65535
+struct flush_func_s
+{
+	/* `flush_func_t' "inherits" from `callback_func_t'.
+	 * The `ff_super' member MUST be the first one in this structure! */
+#define ff_callback ff_super.cf_callback
+#define ff_udata ff_super.cf_udata
+#define ff_ctx ff_super.cf_ctx
+	callback_func_t ff_super;
+	char *ff_name;
+	int ff_type;
```

Comment:
It appears, that setting this to `FF_SIMPLE` is never checked for and only `FF_REMOVE` is used. Please change this to `_Bool ff_remove;` which is initially `0` (false) and set to `1` by `plugin_unregister_flush()`.